sdk/communication/Azure.Communication.NetworkTraversal/src/CommunicationIceServer.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Communication.NetworkTraversal
+{
+    [CodeGenModel(Usage = new[] { "input", "converter" })]
```

Comment:
Nit - consider leaving a comment about how this will allow your model to be serializable and why that's an important customer scenario.